src/Playwright.Tests/BrowserTypeConnectTests.cs

```diff
@@ -31,27 +35,30 @@ namespace Microsoft.Playwright.Tests
     [Parallelizable(ParallelScope.Self)]
     public class BrowserTypeConnectTests : PlaywrightTestEx
     {
+        private BrowserServer _browserServer;
+
+        [SetUp]
+        public void SetUpAsync()
+        {
+            _browserServer = LaunchServer(BrowserType);
```

Comment:
Is there a reason to still have the `LaunchServer` method then? The idea of a `SetUp` method is to have it set everything up. Let's avoid the option of someone launching a separate server, unless we really need it.

Reply:
Have removed method added in `SetUp`